.gitignore

```diff
@@ -55,7 +55,6 @@ typings/
 # hugo files
 .sass-cache
 .hugo_build.lock
-public/
```

Comment:
I don't think we're supposed to check in this folder, since it's the compiled site.  Was there a reason we needed to do this for blog.maxmind.com?

Reply:
I don't remember in particular. I only thought we needed it because this is where we put the CSP stuff (similar commit for the `blog-site`: https://github.com/maxmind/blog-site/pull/499/commits/4010f80d23c2277a36977e683239d9358fbd7dc3)

Reply:
I added this back in. But it isn't clear to me how Cloudflare will be able to read the `_headers` and `_redirects` files since they need to be in the output folder of the project (https://developers.cloudflare.com/pages/configuration/redirects/)

Reply:
You can add it to the `static` folder and it'll be copied over at compile time